src/AddToCalendar.tsx

```diff
@@ -54,13 +54,13 @@ const useOpenState = (initialOpen: boolean): [boolean, OpenStateToggle] => {
 type CalendarRef = HTMLAnchorElement;
 type CalendarProps = {
   children: React.ReactNode;
-  download?: boolean;
+  filename?: string;
   href: string;
 };
 
 const Calendar = React.forwardRef<CalendarRef, CalendarProps>(
-  ({ children, download = false, href }, ref) => (
-    <a ref={ref} download={download} href={href} target="_blank" rel="noopener noreferrer">
+  ({ children, filename = 'download', href }, ref) => (
```

Comment:
I think this is going to make it so that every link downloads, not just the ICS ones. Instead I think you can put the default argument on `Dropdown` and that should fix it (since then you're only passing the filename to the ones you want to be downloadable).

Reply:
Based on my testing it worked fine but I see what you mean, thanks :) I moved the default value in the latest commit (2520be4)